nucypher/network/nodes.py

```diff
@@ -428,8 +428,18 @@ def load_seednodes(self, read_storage: bool = True, retry_attempts: int = 3):
 
     def read_nodes_from_storage(self) -> None:
         stored_nodes = self.node_storage.all(federated_only=self.federated_only)  # TODO: #466
+        domain_inventory = defaultdict(int)
```

Comment:
For lurkers: This change will be removed from this PR before submission for review.